components/typography/Heading.js

```diff
@@ -8,17 +8,38 @@ const factory = (type, defaultElement) => {
     static propTypes = {
       children: PropTypes.node,
       className: PropTypes.string,
+      color: PropTypes.oneOf(['white', 'neutral', 'mint', 'teal', 'violet', 'ruby', 'gold', 'aqua']),
```

Comment:
It looks like we're using this array of colors in a lot of places, maybe we can keep it somewhere separate and reuse it in all components. Should there ever be the need to add a new color, we have to add it in just one place.

Reply:
Yes, we could do that, but not always the same color variations are applicable to our components.